### ISSUE TITLE
New Features for "New scenario" view

### ISSUE THREAD

**Issue by herzogrh** [citysciencelab/scenario-explorer-addon#10]
When creating a new scenario, different new entry types should be added to the addon that can be dynamically filled in: GeoJSON, BBoxes etc. 
Also, new scenario ensembles should be created with uncertainty ranges.

**Comment by KaiVolland** [comment]
It turns out that this is a very complex field and the specification of the inputs can be very wide. I think we have to specify precise conditions on when to show which input field. I also think this should be limited to the three types:
GeoJSON FeatureCollection, GeoJSON (Multi-)Geometry, BBox

**Comment by herzogrh** [comment]
Yes, I think it'd be good to limit the input types to specific ones. BBox should be pretty straight forward. Same goes maybe for file types such as GeoTiffs or csv files. For GeoJSON collections, I would think this is hard to generalize, but I suggest to have a rough implementation without any automatic validation of the geojson scheme so that worst case, an error of the process would be returned.
